prime-dotnet-webapi/Models/EnrolleeAbsence.cs

```diff
@@ -14,7 +14,6 @@ public class EnrolleeAbsence : BaseAuditable, IEnrolleeNavigationProperty
 
         public Enrollee Enrollee { get; set; }
         public DateTime StartTimestamp { get; set; }
-
-        public DateTime EndTimestamp { get; set; }
+        public DateTime? EndTimestamp { get; set; }
```

Comment:
Might need to generate a db migration since current `EnrolleeAbsence` table column `EndTimestamp` is still `not null`

Reply:
That migration will be done in https://github.com/bcgov/moh-prime/pull/1884. I will wait for Nathan's PR to be merged before merging mine.